README.md

```diff
@@ -1,5 +1,9 @@
+[![Travis CI](https://travis-ci.org/anthonydresser/vscode-mssql.svg?branch=dev)](https://travis-ci.org/anthonydresser/vscode-mssql)
```

Comment:
Is there a way to set these up under a Microsoft account?  Ideally we don't take dependencies on accounts owned by individual engineers.